src/formats/AmberNetCDF.cpp

```diff
@@ -114,12 +126,45 @@ void AmberNetCDFFormat::read_step(const size_t step, Frame& frame) {
     }
 }
 
-void AmberNetCDFFormat::read(Frame& frame) {
+template <AmberFormat F>
+void Amber<F>::read(Frame& frame) {
+    if (FormatSpec<F>::one_frame && step_ != 0) {
+        throw format_error("AMBER Restart format only supports reading one frame");
+    }
+
     read_step(step_, frame);
     step_++;
 }
 
-UnitCell AmberNetCDFFormat::read_cell() {
+namespace chemfiles {
```

Comment:
This prevents some error/bug(?) in gcc 4.8.4. Not sure if this is an ideal solution.

Reply:
no, that's the way to do it. The rules of namespaces & template specialization are a bit complex and changed for different standard version.